spec/lib/krikri/indexer_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe Krikri::Indexer do
+  before(:all) do
+    DatabaseCleaner.clean_with(:truncation)
+    create(:krikri_enrichment_activity)
+  end
+
+  enrichment_gen_uri_str = \
+    (RDF::URI(Krikri::Settings['marmotta']['ldp']) /
```

Comment:
Same comment as for [`aggregation_entity_behavior_spec.rb`](https://github.com/dpla/KriKri/pull/131/files#r27392504), without @no-reply's feedback.